packages/alias/README.md

```diff
@@ -140,7 +140,7 @@ Example:
 ```javascript
 // rollup.config.js
 import alias from '@rollup/plugin-alias';
-import resolve from 'rollup-plugin-node-resolve';
+import resolve from '@rollup/plugin-node-resolve';
```

Comment:
I found that the code sample was using the old reference as well. I confirmed that the package.json is using the new npm package

https://github.com/rollup/plugins/blob/875048a13ef7038d2ebd5ffc4515df257d793e6e/packages/alias/package.json#L48-L52

And the new `@rollup/plugin-node-resolve` supports the same `extensions` property